### PR TITLE
Bug 1883479: Add uri value to URI list item and side bar, fix initial scroll to selected item

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
**Fixes**
https://issues.redhat.com/browse/ODC-4911

**Description**
Add the URI link to the list view row for the URI item.
Add the URI link to the header of the URI side panel details (move up from `Resources`)
Fix to scroll to initial selection for topology list view.

**Screenshots**
image

**Browser conformance**: 
- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

/kind bug

/cc @bgliwa01

### CODE REVIEW

**Comment by bgliwa01** [comment]
Could you please change "Uri" in List view "URI"?

**Comment by jeff-phillips-18** [comment]
Updated to `URI`, updated screenshot above.

@divyanshiGupta Please take another look.

**Comment by divyanshiGupta** [comment]
@jeff-phillips-18 the code looks good to me. Waiting for @bgliwa01's approval.

**Review by bgliwa01 — APPROVED** [review]
lgtm! Thanks Jeff

File: frontend/packages/knative-plugin/src/topology/listView/SinkUriListViewNode.tsx
Comment (andrewballantyne) [inline frontend/packages/knative-plugin/src/topology/listView/SinkUriListViewNode.tsx:26]: Nit
```suggestion
      <TypedResourceBadgeCell key="type-icon" kind={item.getResourceKind()} />
```

File: frontend/packages/knative-plugin/src/topology/listView/SinkUriListViewNode.tsx
Comment (andrewballantyne) [inline frontend/packages/knative-plugin/src/topology/listView/SinkUriListViewNode.tsx:27]: If it does not find this, we are back to where we are with this bug 🤔 

Not sure what we could put here though.

**Review by andrewballantyne — APPROVED** [review]

**Pull request merged**
